https://bugs.gentoo.org/494606

From 71f0b7e80ac05eb123b4002536f7c6bec50c4bc9 Mon Sep 17 00:00:00 2001
From: doko <doko@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Fri, 29 Nov 2013 16:53:39 +0000
Subject: [PATCH] 2013-11-29  Matthias Klose  <doko@ubuntu.com>

        * native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c,
        native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c:
        Fix freetype includes.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@205533 138bc75d-0d04-0410-961f-82ee72b054a4
---
 libjava/classpath/ChangeLog.gcj                    |    6 ++++++
 .../gnu_java_awt_peer_gtk_FreetypeGlyphVector.c    |    5 +++--
 .../gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c   |    9 +++++----
 3 files changed, 14 insertions(+), 6 deletions(-)

diff --git a/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c b/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c
index cb7bf43..e74678e 100644
--- a/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c
+++ b/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c
@@ -42,8 +42,9 @@ exception statement from your version. */
 #include <pango/pango.h>
 #include <pango/pangoft2.h>
 #include <pango/pangofc-font.h>
-#include <freetype/ftglyph.h>
-#include <freetype/ftoutln.h>
+#include <ft2build.h>
+#include FT_GLYPH_H
+#include FT_OUTLINE_H
 #include "jcl.h"
 #include "gdkfont.h"
 #include "gnu_java_awt_peer_gtk_FreetypeGlyphVector.h"
diff --git a/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c b/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
index 771b23e..cfd988c 100644
--- a/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
+++ b/libjava/classpath/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
@@ -39,10 +39,11 @@
 #include <pango/pango.h>
 #include <pango/pangoft2.h>
 #include <pango/pangofc-font.h>
-#include <freetype/ftglyph.h>
-#include <freetype/ftoutln.h>
-#include <freetype/fttypes.h>
-#include <freetype/tttables.h>
+#include <ft2build.h>
+#include FT_GLYPH_H
+#include FT_OUTLINE_H
+#include FT_TYPES_H
+#include FT_TRUETYPE_TABLES_H
 #include "gdkfont.h"
 #include "gtkpeer.h"
 #include "gnu_java_awt_peer_gtk_GdkFontPeer.h"
-- 
1.7.1

